### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by AmirMehrabi** [aloncarmel/devunityatom#1]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.7
**System**: Ubuntu 14.04.3
**Thrown From**: [devunity](https://github.com/aloncarmel/devunityatom) package, v2.2.0
**User**: root

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /root/.atom/packages/devunity/lib/devunity.coffee:410

TypeError: undefined is not a function
    at DevunityView.getLanguage (/root/.atom/packages/devunity/lib/devunity.coffee:681:31)
    at DevunityView.start (/root/.atom/packages/devunity/lib/devunity.coffee:551:25)
    at atom-workspace.<anonymous> (/root/.atom/packages/devunity/lib/devunity.coffee:467:24)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app.asar/src/command-registry.js:156:19)
    at EventEmitter.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:70:30)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:166:7)
```

### Commands

```
     -0:11.6.0 atom-beautify:beautify-editor (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:00.8.0 devunity:start (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {},
  "devunity": {
    "apikey": "anonymous"
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.11
color-picker, v2.0.11
devunity, v2.2.0
language-blade, v0.15.0
minimap, v4.12.3

# Dev
No dev packages
```

**Comment by yamsellem** [comment]
Still an issue with atom 1.0.11.

**Comment by aloncarmel** [comment]
Thanks, sorry for that, im on it.

**Comment by aloncarmel** [comment]
Fixed in 3.0, update your package and thanks for the reporting!
